models/staging/jaffle_shop/src_jaffle_shop.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+
+sources:
+    - name: jaffle_shop
+      description: A clone of a Redshift application database.
```

Comment:
oh nice!  You are already moving this into your own folder system here AND a specific YML file for the source.  I'll be modeling this a little in `polish your dbt project` today since we only alluded to this on Day One.

I personally like the naming convention that you are using here (`src_jaffle_shop.yml`) because it is super easy to search for.  I have seen other folks just call this `source.yml` in the particular source folder, jaffle_shop in this case.